HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+0.5.6 (2023-01-03)
```

Comment:
Instead of this line, write `Not released yet`. The version and date will be added by maintainer, just before creating a new version.